aten/src/ATen/native/cuda/Loss.cu

```diff
@@ -3,6 +3,7 @@
 #include <ATen/Dispatch.h>
 #include <ATen/cuda/CUDAApplyUtils.cuh>
 
+#define EPSILON 1e-12
```

Comment:
nit: in C++ there ~essentially shouldn't be any reason to define constants this way; just use constexpr.